src/brackets/index.js

```diff
@@ -4,6 +4,63 @@
  * @param {string} str The string of brackets.
  * @returns {"valid" | "invalid"} Whether or not the string is valid.
  */
-function isValid(str) {}
+/*
+function isValid(str) 
+{ 
+    let brackets = "[]{}()<>";
+    let stack = [];
+
+
+    for(let bracket of input) 
+    {
+     let bracketsIndex = brackets.indexOf(bracket)   
+   
+    if(bracketsIndex % 2 ===0) 
+    {
+       stack.push(bracketsIndex + 1)
+   
+    if (stack.pop() !== bracketsIndex) 
+    {
+        return "invalid"
+    }
+   } 
+
+   }
+}
+*/
+function isValid(str) {
+
+    if (str.length <= 1)
+      return "invalid"
+    // MOB mathching opening Bracket
+    //cB closing Brackets
+    //oB Opening Brackets
+    let mOB, ch
```

Comment:
Rather than use comments to describe what your variables mean, it makes sense to just name your variables that. Also, stick to one variable declaration per line.

``` js
let matchingOpeningBracket;
let closingBrackets;
```

are better variable names.